diagnostic.md

```diff
@@ -100,11 +130,23 @@ object.
 ```
 
 ```sh
-# your answer here
+curl --include --request PATCH "http://www.example.com/donuts/42" \
+  --header "Content-Type: application/json" \
+  --data '{
+    "name": "Krüller"
+  }'
 ```
 
 Write an AJAX request to change the donut on the server using JSON.
 
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function(){
+  return $.ajax({
+    url: 'http://www.example.com/donuts/42',
+    method: 'PATCH',
+    '{
+      "name": "Krüller"
+    }',
+  });
+};
```

Comment:
Again, missing `data` parameter key here. A better solution would be to take the id in as an argument, like this:
```
let changeDonut = function (donutID) {
  return $.ajax({
    url: 'http://example.com/donuts/' + donutID,
    method: 'PATCH',
    data: {
      "donut" {
        "name": "Krüller",
      }
    }
  });
};
```